defs/constants.bzl

```diff
@@ -0,0 +1,3 @@
+"""Constants to be used by rules."""
+# This file is not formatted with Buildifier on purpose to allow custom formatting
+EXAMPLES_DG_JSON="examples/dg.json"
```

Comment:
Excluded file from the formatting -- to be able to have custom formatting of the source code.